packages/protobuf/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 1.0.14-beta.2
+
+-   chore(suite): unused package dependencies removed (f7907e1496)
+-   chore(suite): depcheck enabled (2206f19f2e)
```

Comment:
again, both messages are scoped to suite although they are changeing protobuf package, who was the culprit of this 😠

Reply:
I am going to add it to the code improvements proposals and think if there is a way to enforce it. :thinking: